### PR TITLE
Added iOS Universal User Agent.

### PR DESCRIPTION
**Pull request by samuli-hiltunen-somedia**
I've added support for iOS Universal User Agent tag, so please review and let me know - or just add it, please. :)

### CODE REVIEW

**Comment by samuli-hiltunen-somedia** [comment]
What needs to be done with this one to move on, please? 

"Merging can be performed automatically with 1 approving review."

File: src/isMobile.ts [hunk 1]
```diff
@@ -79,6 +81,7 @@ export default function isMobile(userAgent?: string): isMobileResult {
       phone: match(appleIphone) && !match(windowsPhone),
       ipod: match(appleIpod),
       tablet: !match(appleIphone) && match(appleTablet) && !match(windowsPhone),
+      universal: match(appleUniversal),
       device:
         (match(appleIphone) || match(appleIpod) || match(appleTablet)) &&
         !match(windowsPhone),
```
Comment (OlympicLarry) [inline src/isMobile.ts]: apple.device should include universal

File: src/isMobile.ts [hunk 2]
```diff
@@ -129,6 +132,7 @@ export default function isMobile(userAgent?: string): isMobileResult {
   };
 
   result.any =
+    result.apple.universal ||
```
Comment (OlympicLarry) [inline src/isMobile.ts]: not needed when apple.universal is included in apple.device

File: src/isMobile.ts (hunk 1 above)
Comment (kaimallea) [reply src/isMobile.ts:87]: +1, should be added to `device`

File: src/isMobile.ts (hunk 2 above)
Comment (kaimallea) [reply src/isMobile.ts:135]: +1

**Comment by kaimallea** [comment]
I'll merge and then apply @OlympicLarry 's suggestions that I agree with

**Pull request merged**